reflex/config.py

```diff
@@ -223,24 +223,32 @@ def __init__(self, *args, **kwargs):
 
         super().__init__(*args, **kwargs)
 
-        # set overriden class attribute values as os env variables to avoid losing them
+        # Load config values from env_path and override os_env if override_os_env is True
+        if self.env_path is not None and os.path.isfile(self.env_path):
+            load_dotenv(self.env_path, override=self.override_os_envs)  # type: ignore
+            # Recompute constants after loading env variables
+            importlib.reload(constants)
+            # Recompute instance attributes
+            self.recompute_field_values()
+
+        # Set class attributes not present in os_env in os_env
         for key, value in dict(self).items():
             key = key.upper()
             if (
                 key.startswith("_")
                 or key in os.environ
                 or (value is None and key != "DB_URL")
             ):
+                if key in os.environ and key == "ENV":
+                    env_values: Dict[str, constants.Env] = {
+                        "Env.DEV": constants.Env.DEV,
+                        "Env.PROD": constants.Env.PROD,
```

Comment:
i think we should just expect the user to pass "dev" or "prod" here (the value of the enum item), not the dot-qualified string.

Reply:
> i think we should just expect the user to pass "dev" or "prod" here (the value of the enum item), not the dot-qualified string.

Right, will do